zombies.js

```diff
@@ -79,18 +93,51 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+class Player {
+  constructor(name, health, strength, speed) {
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this._name = name;
+    this._health = health;
+    this._strength = strength;
+    this._speed = speed;
```

Comment:
name, health, speed, and strength did not need to be private. Also, the get name, get itemName methods below are also unneeded.